libs/ledger-live-common/src/exchange/swap/completeExchange.ts

```diff
@@ -130,9 +133,26 @@ const completeExchange = (
 
         if (unsubscribed) return;
 
+        /**
+         * LiFi swap transactions are frequently failing on-chain.
+         * The failures are due to "out of gas" errors. This indicates the gas limit estimated for the transaction is too low.
+         * Manually resubmitting a failed transaction with a higher gasLimit allows it to succeed.
+         * The root cause is that the partner (LiFi) provides "don’t give accurate gasLimit" estimations.
```

Comment:
It is this and also our explorers are somehow under-estimating the tx, or the gasLimit changes and get increased in the next few blocks. Just a nitpick for the comment since we're updating our explorers value here and not LiFi's